README.md

```diff
@@ -46,23 +78,47 @@ task has bundled the following required tasks:
    `release.properties` file, plus the version, in the task `writeLibraryProperties`.
    4. within the `releaseProcessingLib` task, the `release` folder is created, jars of the library and
    dependencies are copied, and the `library.properties` file is copied. Also, a zip file is made.
-3.`copyToLocalProcessing` will copy the contents of the release folder into the relevant
-folder in the sketchbook folder, a folder that Processing creates that holds your installed
-libraries, among other things. This installs your library into Processing, and you can test
-the release with your examples.
 
-### Using GitHub Actions for Library Release
+
+#### Using GitHub Actions for Library Release
 You can automatically distribute the library using GitHub Actions. Libraries are distributed in `.zip` format to the [releases section](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) of the GitHub repository. The trigger to execute the release workflow is pushing a git tag that starts with `v`. For example:
 ```sh
 git tag v0.0.1
 git push origin v0.0.1
 ```
 
+
+
+### Creating your website
+A requirement for having your library listed officially is to have a stable website.  Library developers
+have clued us into [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/),
+as a simple and easy to use way of creating content. If your library is on Github, then you can take
+advantage of Github Pages to deploy and host the website. We configure this template for this scenario.
+
+MkDocs is Python-based, so if you are not already a Python user, hopefully your OS already has a system
+Python installed. You can check by running `python --version` in a terminal. If you do not already have
+Python installed, or do not have Python 3, you will need to install Python 3 as a prerequisite to these
+instructions.
+
+1. Install [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+   Please refer to the documentation. For example, run `pip install mkdocs-material` in a terminal.
+2. We already created a very barebones website in the template. you can
```

Comment:
Same, let's have the default path be just publishing through editing the repository, then if people want live-preview then can go through installing MKDocs themselves.